coord/src/dist_aggregate.c

```diff
@@ -470,9 +470,7 @@ static void rpnetFree(ResultProcessor *rp) {
     rm_free(nc->shardsProfile);
   }
 
-  if (nc->current.root) {
-    MRReply_Free(nc->current.root);
-  }
```

Comment:
Why we check for not NULL before? There is no case it will be NULL?

Reply:
It might be, but it is also checked inside the function

Reply:
amm oK